application/templates/components/filter-checkboxes/macro.jinja

```diff
@@ -1,4 +1,14 @@
 {% macro dlFilterCheckboxes(params) %}
+<script type="module">
+  import FilterCheckboxesController from '/static/javascripts/FilterCheckboxesController.js'
+  const initFilterCheckbox = () => {
+    let checkboxContainer = document.getElementById('checkboxes-{{ params.uniqueID }}');
+    let searchBox = document.getElementById('input-{{ params.uniqueID }}');
+    return new FilterCheckboxesController(checkboxContainer, searchBox)
+  }
+  initFilterCheckbox();
```

Comment:
Consider making `params.uniqueID` a parameter of the function so that the dependencies are made explicit.